partiql-parser/src/scanner.rs

```diff
@@ -208,6 +243,11 @@ impl<'val> PartiQLScanner<'val> {
                     _ => return number_pair.unexpected(),
                 }
             }
+            Rule::Dot_ => Content::Dot,
+            Rule::Star_ => Content::Star,
+            Rule::Parameter => Content::Parameter,
```

Comment:
Is there a reason tests haven't been added for Dot, Star, and Parameter?

Reply:
Good catch, got so caught up in my other punctuation tests especially around potentially ambiguous cases that I forgot. will add.